collect_env.py

```diff
@@ -0,0 +1,469 @@
+from __future__ import print_function
```

Comment:
This should not be here right (the whole file)?

Reply:
ah, good catch. I'll remove it